model/otel/metrics.yaml

```
@@ -110,3 +110,98 @@ groups:
           recommended: when applicable
       - ref: error.type
         examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
+
+  - id: metric.otel.sdk.log.count
+    type: metric
+    metric_name: otel.sdk.log.count
+    stability: development
+    brief: "The number of logs submitted to enabled SDK Loggers"
+    instrument: counter
+    unit: "{LogRecord}"
+
+  - id: metric.otel.sdk.processor.log.queue.size
+    type: metric
+    metric_name: otel.sdk.processor.log.queue.size
+    stability: development
+    brief: "The number of LogRecords in the queue of a given instance of an SDK log processor"
+    note: |
+      Only applies to LogRecord processors which use a queue, e.g. the SDK Batching LogRecord Processor.
+    instrument: updowncounter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+
+  - id: metric.otel.sdk.processor.log.queue.capacity
+    type: metric
+    metric_name: otel.sdk.processor.log.queue.capacity
+    stability: development
+    brief: "The maximum number of LogRecords the queue of a given instance of an SDK LogRecord processor can hold"
+    note: |
+      Only applies to LogRecord processors which use a queue, e.g. the SDK Batching LogRecord Processor.
+    instrument: updowncounter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+
+  - id: metric.otel.sdk.processor.log.processed_logs
+    type: metric
+    metric_name: otel.sdk.processor.log.processed_logs
+    stability: development
+    brief: "The number of LogRecords for which the processing has finished, either successful or failed"
+    note: |
+      For successful processing, `error.type` MUST NOT be set. For failed processing, `error.type` must contain the failure cause.
+      For the SDK Simple and Batching LogRecord Processor a LogRecord is considered to be processed already when it has been submitted to the exporter,
+      not when the corresponding export call has finished.
+    instrument: counter
+    unit: "{LogRecord}"
+    attributes:
+      - ref: otel.component.type
+      - ref: otel.component.name
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching LogRecord Processors MUST use `queue_full` for LogRecords dropped due to a full queue.
+        examples: ["queue_full"]
+
+  - id: metric.otel.sdk.exporter.log.inflight_logs
+    type: metric
+    metric_name: otel.sdk.exporter.log.inflight_logs
+    stability: development
+    brief: "The number of LogRecords which were passed to the exporter, but that have not been exported yet (neither successful, nor failed)"
+    note: |
+      For successful exports, `error.type` MUST NOT be set. For failed exports, `error.type` must contain the failure cause.
+    instrument: updowncounter
+    unit: "{LogRecord}"
```

I think snake case has been the existing convention for units:

```suggestion
    unit: "{log_record}"
```

---

Fixed in [1e1af04](https://github.com/open-telemetry/semantic-conventions/pull/1921/commits/1e1af04e42d5e39bafb5b5da586566e6ff038916)